cji_smoke_test.sh

```diff
@@ -66,6 +67,14 @@ if [ "$IQE_SELENIUM" = "true" ]; then
     SELENIUM_ARG=" --selenium "
 fi
 
+ENV_VAR_ARGS=""
+if [ -z "$IQE_ENV_VARS" ]; then
```

Comment:
Doesn't this mean we'll only process this block if IQE_ENV_VARS is empty? Shouldn't it be like:

```
if [ -n "$IQE_ENV_VARS" ]; then
    IFS=',' read -ra values_array <<< "$IQE_ENV_VARS"
    for i in "${values_array[@]}"; do
        ENV_VAR_ARGS="${ENV_VAR_ARGS} --env-var $i"
    done
fi
```

Reply:
Good catch, yes I had `[ ! -z "$IQE_ENV_VARS" ]` written in my test script :grimacing: